views/js/review/plugins/navigation/review-panel/tpl/fizzyPanel.tpl

```diff
@@ -20,3 +20,24 @@
     </footer>
 {{/if}}
 </div>
+{{#if replaceIcons}}
+<style>
+    {{#each replaceIcons}}
+        .buttonlist-items .buttonlist-item.{{@key}} .buttonlist-btn .buttonlist-score-badge {
+            background-color: {{background}};
+            color: {{color}};
+        }
+        .item-answer.show-correct.{{@key}} .icon {
+            color: {{color}};
+        }
+        {{#if code}}
+            .buttonlist-item.{{@key}} .buttonlist-score-badge .icon-{{@key}}:before {
+                content: "{{code}}";
+            }
+            .item-answer .icon-{{@key}}:before {
+                content: "{{code}}";
+            }
+        {{/if}}
+    {{/each}}
+</style>
+{{/if}}
```

Comment:
**quibble:** Please avoid this kind of construction. It prevents custom themes since the inline style supersedes the imported style.

Reply:
I agree, but first request was that customer need to update icons by configs, because they didn't know what icons should be. Later they provided icons and colors. That's why configuration and not custom theme.
But I will clean this and move to custom theme.